wix/Build.OpenJDK_generic.cmd

```diff
@@ -67,7 +71,7 @@ REM
 
 REM Cultures: https://msdn.microsoft.com/de-de/library/ee825488(v=cs.20).aspx
 SET PRODUCT_SKU=OpenJDK
-SET PRODUCT_VERSION=%PRODUCT_MAJOR_VERSION%.%PRODUCT_MINOR_VERSION%.%PRODUCT_MAINTENANCE_VERSION%.%PRODUCT_PATCH_VERSION%
+SET PRODUCT_VERSION=%PRODUCT_MAJOR_VERSION%.%PRODUCT_MINOR_VERSION%.%PRODUCT_MAINTENANCE_VERSION%.%PRODUCT_PATCH_VERSION%.%PRODUCT_BUILD_NUMBER%
```

Comment:
Didn't we must add a "+" or a "b" as "**b**uild" here before the PRODUCT_BUILD_NUMBER to stay consitant ..
I think Windows can handle "+" in filename and PRODUCT_VERSION is now only used in msi file name OUTPUT_BASE_FILENAME

Reply:
Before this PR, the "PRODUCT_PATCH_VERSION" is actually the build number. It's named wrong, and this PR fixes that.
Adding a plus sign or a "b" is an additional behaviour change.